mavros/param/config.yaml

```diff
@@ -0,0 +1,207 @@
+mavros_node:
+  ros__parameters:
+    fcu_url: "udp://:14550@"
+    gcs_url: ""
+    tgt_system: 1
+    tgt_component: 1
+
+###################
+### plugin params ###
+###################
+
+# actuator_control None
+# adsb None
+
+# altitude:
+#   ros__parameters: 
+#   frame_id: map
+
+# cam_imu_sync None
+# camera None
+
+# cmd:
+#   ros__parameters:
+#     use_comp_id_system_control: true
+#     command_ack_timeout: 5000ms
+
+# companion_process_status None
+# debug_value None
+# distance_sensor None
+# esc_status None
+# esc_telemetry None
+# fake_gps None
+# ftp None
+
+# geofence:
+#   ros__parameters:
+#     pull_after_gcs: true
+#     use_mission_item_int: true
+
+# global_position:
+#   ros__parameters:
+#     frame_id: map
+#     child_frame_id: base_link
+#     rot_covariance: 99999.0
+#     gps_uere: 1.0
+#     use_relative_alt: true
+#     tf:
+#       send: false
+#       frame_id: map
+#       global_frame_id: earth
+#       child_frame_id: base_link
+
+# gps_input None
+# gps_rtk None
+# gps_status None
+# hil None
+# home_position None
+
+# imu:
+#   ros__parameters:
+#     frame_id: true
+#     linear_acceleration_stdev: 0.0003
+#     angular_velocity_stdev: 0.02 * (M_PI / 180.0)
+#     orientation_stdev: 1.0
+#     magnetic_stdev: 0.0
+
+# landing_target None
+# gps_rtk None
+
+# local_position:
+#   ros__parameters:
+#     frame_id: map
+#     tf:
+#       send: false
+#       frame_id: map
+#       child_frame_id: base_link
+
+# log_transfer None
+# mag_calibration_status None
+# manual_control None
+# mocap_pose_estimate None
+# mount_control None
+# nav_controller_output None
+# obstacle_distance None
+
+# odometry:
```

Comment:
Parameter name would be: `<UAS-name>/<plugn-name>`. E.g. `mavros/odometry`.

Reply:
change <plugn-name> --> mavros/<plugn-name>